keps/sig-network/2079-network-policy-port-range/kep.yaml

```diff
@@ -18,7 +18,7 @@ stage: alpha
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.21"
+# latest-milestone: "v1.21"
```

Comment:
@rikatz FYI, PRR approval will need to be part of the PR that sets this back to 1.21.

Reply:
@johnbelamaric @wojtek-t ok, will review the KEP questionaire about the PRR. PR for the feature is open https://github.com/kubernetes/kubernetes/pull/97058 

Need to submit a new PR for the KEP with the questions revisited, right?

Reply:
Correct.
Or rather - I see that you already have filled some PRR sections (didn't look deep enough if all needed for Alpha, but potentially). So basically, you just need to have that reviewed by the PRR approvers.

Reply:
@wojtek-t completed some more questions there.

If you don't mind being the PRR approver there, I can change the kep.yaml to have you as the prr-approver :) 

Thanks!

Reply:
I'm happy to review but I'm OOO for the next 2.5 weeks - can review in the second full week of January - please assign the PR to me directly.

Reply:
No rush :) 

I'll assign you to the PR and mark you as the PRR reviewer in kep.yaml.

Thanks and happy new year